src/ape/pytest/contextmanagers.py

```diff
@@ -1,26 +1,141 @@
+import re
+from collections import deque
 from typing import Optional, Type
 
-from ape.exceptions import ContractLogicError, TransactionError
+import ape
+from ape.exceptions import APINotImplementedError, ContractLogicError, TransactionError
 
 
 class RevertsContextManager:
-    def __init__(self, expected_message: Optional[str] = None):
+    def __init__(self, expected_message: Optional[str] = None, dev_message: Optional[str] = None):
+        if dev_message is not None and expected_message is not None:
```

Comment:
`brownie` does let you use both

Reply:
It was noted in the [original issue discussion](https://github.com/ApeWorX/ape/issues/251) that it should be mutually exclusive - is this somewhere where we can/should break with Brownie's implementation?

cc @NotPeopling2day @fubuloubu

Reply:
It may be a mistake that it was written that way! That is my impression. I don't see a technical reason why you couldn't assert both.

Reply:
Yeah and if we do allow for both, I think the only code change needed here will just be removing the `ValueError` case and then further down just using two `if` statements for message checking instead of `if/else`

Reply:
One use-case I can think of for wanting both is to expect a certain shorter message as the actual message and a different more verbose message as the dev string.